### PR TITLE
Added arabic README

### PR DESCRIPTION
**Pull request by eymeen**

### CODE REVIEW

**Comment by YasinDehfuli** [comment]
Thanks Dear @eymeen.
You need to do some changes to improve your work:
1. Add your Arabic Button in end of the list or next to English button.
2. Don't translate the Arabic translate buttons and let it stay English like other pages.
3. We need your full translate - please add steps-ar folder and translate all steps of arabic language too.

File: README.ar.md [hunk 1]
```diff
@@ -0,0 +1,376 @@
+<!-- <div align="center"> -->
+
+## اللغات المتوفرة
+
+<div align="center">
+  
+[![ar](https://img.shields.io/badge/Lang-العربية-black.svg)](https://github.com/4xmen/Get-Github-Achievements/blob/master/README.ar.md)
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+[![en](https://img.shields.io/badge/Lang-الإنجليزية-blue.svg)](https://github.com/4xmen/Get-Github-Achievements/blob/master/README.md)
```
Comment (YasinDehfuli) [inline README.ar.md:9]: Please change all buttons names to English.

File: README.ar.md [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+</div>
+<hr>
+
+## كيفية الحصول على إنجاز Yolo
+
+### إن Yolo هو واحد من أجمل الإنجازات في GitHub ، والتي يمكن أن تمنح تأثيرًا جميلًا جدًا على ملف GitHub الخاص بك. للحصول على هذا الإنجاز ، تحتاج إلى دمج طلب استحقاق بدون مراجعة.
+
+#### - إذا كنت بحاجة إلى المزيد من المساعدة ، انقر على زر "كيفية الحصول" للحصول على برنامج تعليمي خطوة بخطوة حول كيفية الحصول على هذا الإنجاز.
+
+<div align="center"><a href="assets/steps/README.yolo.md"><img src="assets/img/btn.png" alt=""></a></div>
```
Comment (YasinDehfuli) [inline README.ar.md:106]: If you can, please, Change the How to get button to Arabic for better interview

Comment (YasinDehfuli) [inline README.ar.md:106]: Please create steps-ar folder and translate all how to get steps to Arabic and link it in your Arabic main translate for complete your job.

File: README.ar.md [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+<hr>
+
+<br>
+
+<div align="center">
+<a href="assets/steps/README.galaxy-brain.md">
```
Comment (YasinDehfuli) [inline README.ar.md:112]: don't forget to link your steps-ar translate pages here too.

File: README.ar.md [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+#### يمكنك العثور على مزيد من المعلومات حول شارات GitHub تحت هذا [الرابط](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile).
+
+<!-- </div> -->
+<br>
+
+####
+
+####          
+
+
```
Comment (YasinDehfuli) [inline README.ar.md:376]: After add steps-ar and change fix bugs and do changes, we will see you as our contributions.

Thanks dear @eymeen.

See you soon as our contributors.

**Review by YasinDehfuli — CHANGES_REQUESTED** [review]
Please do these changes. And translate steps to Arabic too.

Good luck And see you soon dear @eymeen 

- @YasinDehfuli

File: README.ar.md (hunk 4 above)
Comment (eymeen) [reply README.ar.md:376]: Done my man

File: assets/btn-ar.png
Comment (YasinDehfuli) [inline assets/btn-ar.png:1]: Please change the btn-ar.png location to https://github.com/4xmen/Get-Github-Achievements/tree/main/assets/img 

your btn-ar.png is currently in assets, move it to ⇉ assets/img file

**Review by YasinDehfuli — CHANGES_REQUESTED** [review]

**Pull request merged**